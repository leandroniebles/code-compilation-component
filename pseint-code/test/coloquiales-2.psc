proceso errores
  escribir es cero
  escribir x es divisible por
finproceso