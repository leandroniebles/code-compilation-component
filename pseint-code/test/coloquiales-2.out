=== Line 2: SynError 317
=== Line 3: SynError 318
